toolchain/check/testdata/basics/no_prelude/raw_identifier.carbon

```diff
@@ -2,10 +2,6 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-// ARGS: compile --no-prelude-import --phase=check --dump-sem-ir %s
-//
-// Check that the command-line flag to dump textual IR works.
```

Comment:
This seems to lose this check? Or is this maybe copy-pasted from some other test?

Reply:
I see it below in the `textual_ir` test. Makes sense.